### ISSUE TITLE
Assuming "x" is within bounds for index function

### ISSUE THREAD

**Issue by niazim3** [smiths/caseStudies#133]
Under the MIS for the SeqServices module

![image](https://user-images.githubusercontent.com/28247301/43144970-54c1a9f4-8f2c-11e8-825c-3855adf710b4.png). There is no `isInBounds` function defined in the MIS, so what bounds is this assumption referring to? If it's referring to the bounds that are defined below the assumption (i.e. X<sub>i</sub> <= x < X<sub>i+1</sub>), then I don't think it should be an assumption since the definition clearly states the boundaries there.

**Comment by smiths** [comment]
You are correct that this is ambiguous.  In bounds in this case means X[0] <= x <= X[|X|-1].  It isn't possible to return an index for x if it lies outside of the data in X.  Can you please update the MIS to include this clarification.

**Comment by niazim3** [comment]
Makes sense, I can add this function from an assignment from 2ME3: 
![image](https://user-images.githubusercontent.com/28247301/43146772-cc72996a-8f2f-11e8-989c-50c12fa0353b.png) 😄

**Comment by smiths** [comment]
Yes, you can add this function.  Good idea.
